blog/en/blog/2023/01/02/accessing_apisix-dashboard_from_everywhere_with_keycloak_authentication.md

```diff
@@ -0,0 +1,1036 @@
+---
+title: "Accessing Apisix-dashboard from everywhere with Keycloak authentication"
```

Comment:
Hi, please use `APISIX` instead of `Apisix` and check all other places. :) We only use `Apisix` in Ingress Resources like `ApisixRoute`.

Reply:
@juzhiyuan  Please help: this is the first time I do a revision in github
Where have I to do the changes? 
Here: https://github.com/MirtoBusico/apisix-website/blob/master/blog/en/blog/2023/01/02/accessing_apisix-dashboard_from_everywhere_with_keycloak_authentication.md

And after the modifications simply commiy?

Reply:
I suppose I have catched alla APISIX occurrencies :)